### PR TITLE
Removed alpha kubeconfig command

### PR DESCRIPTION
**Pull request by hpandeycodeit**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

This removes the `alpha kubeconfig ` command as discussed in #[1381](https://github.com/kubernetes/kubeadm/issues/1381)

**Which issue(s) this PR fixes**:

Fixes # [1381](https://github.com/kubernetes/kubeadm/issues/1381)

```release-note
NONE
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,3 +130,4 @@ zz_generated_*_test.go
 
 # generated by verify-godeps.sh
 vendordiff.patch
+/.metadata/
```
Comment (neolit123) [inline .gitignore]: we shouldn't touch the .gitignore file as part of this PR.

**Review by neolit123 — COMMENTED** [review]
thank you for this PR @hpandeycodeit !

/priority important-longterm

before running CI tests please call `./hack/update-bazel.sh` and include the changes in the same single commit. (`git push -f` works for that).

also see my note about `.gitignore`.
thanks.

@kubernetes/sig-cluster-lifecycle-pr-reviews

File: .gitignore (hunk 1 above)
Comment (hpandeycodeit) [reply .gitignore]: I have reverted it.

**Comment by hpandeycodeit** [comment]
@neolit123 Thanks! I have updated the commit with the `./hack/update-bazel.sh`

**Review by neolit123 — APPROVED** [review]
thanks for the update.
/ok-to-test
/approve

**Review by yagonobre — COMMENTED** [review]
Thanks @hpandeycodeit
/lgtm

**Comment by yagonobre** [comment]
@hpandeycodeit I found a minor typo on release note `depricated` -> `deprecated`
/hold

**Comment by hpandeycodeit** [comment]
@yagonobre Thanks for pointing it out. I have updated the release note.

**Comment by hpandeycodeit** [comment]
Thank you @yagonobre @neolit123 !! :)

**Comment by neolit123** [comment]
@hpandeycodeit 
we had to revert this PR since some users complained about the removal of the command.
we are going to look at this command again in 1.15.

can you please change the release note to `NONE`

instead of:
> "kubeadm alpha kubeconfig" command is deprecated. 
> "kubeadm init phase kubeconfig admin" can be used instead to create kubeconfig files

thanks!

**Comment by hpandeycodeit** [comment]
@neolit123 done. So that means this will remain for a while or we are completely reverting it back?

**Comment by neolit123** [comment]
thanks.
it will remain for a while and we need to decide what to do with it in the 1.15 cycle as it's under "alpha".

**Pull request merged**